### DISCUSSION TITLE
How to add Vuestic Plugin

### DISCUSSION THREAD

**Discussion by harblaith7** [nuxt/nuxt#18142] [Questions] [answered]
I want to add a UI Vue library to my Nuxt3 and I choose Vuestic. In order to do this, I created a `plugins` directory and inside a `vuestic.ts` file. In that file I had the following code 

```
import { VuesticPlugin } from "vuestic-ui";

import { defineNuxtPlugin } from "#app";

export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(VuesticPlugin);
});
```

However, when I do 

```vue
  <va-alert border="top" class="mb-4">
      You read this important alert message.
  </va-alert>
```

It errors out. Any solutions

**Comment by wobsoriano** [comment, marked as answer]
https://github.com/nuxt/framework/discussions/1869
